lib/utils/sendNotifications.js

```diff
@@ -0,0 +1,27 @@
+const users = require('../../lib/modelManagers/followModel');
+const sendVerficationMail = require('../../lib/utils/emailService/emailNotification');
+const articleModelManager = require('../../lib/modelManagers/articlemodel');
+const user = require('../../lib/modelManagers/userProfileModel');
+
+const getUsersToNotify = async (id, title, article) => {
+  let UsersToNotify = await users.getUserFollowees(id);
+  const author = await articleModelManager.getArticlesby('authorId', id, 2, 0);
+  const authorName = `${author[0].dataValues.articleAuthor.dataValues.firstname} ${author[0].dataValues.articleAuthor.dataValues.lastname}`;
+  UsersToNotify.rows.forEach(async (element) => {
+    UsersToNotify = await user.getUserDetails('UserId', element.dataValues.following.dataValues.id);
```

Comment:
Why are you mutating `UsersToNotify` while you are iterating through its row attribute?

Reply:
I have fixed this, by using a new variable for each user returned from the iteration